docs/docs-content/troubleshooting/generate-har-files.md

```diff
@@ -0,0 +1,178 @@
+---
+sidebar_label: "Generate HAR Files"
+title: "Generate and Sanitize HTTP Archive Files"
+description: "Learn how to generate and sanitize HAR files to speed up issue resolution."
+sidebar_position: 70
+tags: ["troubleshooting", "har", "support"]
+---
+
+When you report issues with Pallete to Spectro Cloud Support, we may ask you to generate an HTTP Archive (HAR) file to help us identify and resolve those issues.
+
+:::caution
+HAR files contain all network activity for a specific page, including sensitive and confidential information, such as API keys, secrets, cookies, passwords, and more. Before sending your HAR files to us, sanitize the sensitive data. We do not recommend using any third-party tools for HAR sanitization, as they can overlook some details.
+:::
+
+This guide explains how you can generate HAR files in Chrome, Safari, and Firefox and sanitize the sensitive data they contain.
+
+## Prerequisites
+
+- A browser of choice (we recommend Chrome, Safari, or Firefox).
+- A text editor of choice (we recommend [Visual Studio Code](https://code.visualstudio.com/) or [Sublime](https://www.sublimetext.com/)).
+
+## Generate and Sanitize HARs
+
+### Generate HAR Files
+
+<Tabs>
+  <TabItem value="chrome" label="Chrome" default>
+    1. Open the page with issues in Chrome.
+    2. On your keyboard, open Developer Tools:
+       - For Windows and Linux, press **F12** or **Ctrl + Shift + I**.
+       - For Mac, press **Fn + F12** or **Cmd + Option + I**.
+
+       For alternative methods to open Developer Tools, refer to [Open Chrome DevTools](https://developer.chrome.com/docs/devtools/open).
+    3. In Developer Tools, select the **Network** tab, and, on the **Network** pane:
+       - If the leftmost icon displays a circle within a circle, click it to start recording network activity.
+       - Click the **Clear network log** icon that follows it to clear the current network log.
```

Comment:
Is this a sequence of steps that user needs to take? if so, we should use numbered lists instead (it would probably be a, b, c).